content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -1,52 +1,114 @@
-that controls how API data is encrypted in etcd.
-The configuration is provided as an API named
-[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). `--encryption-provider-config-automatic-reload` boolean argument determines if the file set by `--encryption-provider-config` should be automatically reloaded if the disk contents change. This enables key rotation without API server restarts. An example configuration is provided below.
+that specifies a path to a configuration file. The contents of that file, if you specify one,
+control how Kubernetes API data is encrypted in etcd.
+
+The format of that configuration file is YAML, representing a configuration API kind named
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+You can see an example configuration
+in [Encryption at rest configuration](#understanding-the-encryption-at-rest-configuration).
+
+Check whether the `kube-apiserver` process is running with the `--encryption-provider-config`
+command line argument set to `true`. If it is not, you do not have encryption at rest enabled.
+Otherwise, you need to perform further checks to confirm the situation.
+
+If `--encryption-provider-config` is set, check which resources (such as `secrets`) are
+configured for encryption, and what provider is used.
+Make sure that the preferred provider for that resource type is **not** `identity`; you
+only set `identity` (_no encryption_) as default when you want to disable encryption at
+rest.
+Verify that the first-listed provider for a resource is something **other** than `identity`,
+which means that any new information written to resources of that type will be encrypted as
+configured. If you do see `identity` as the first-listed provider for any resource, this
+means that those resources are being written out to etcd without encryption.
+
+{{< note >}}
+It may not be enough to see that encryption at rest is active for new data.
+
+If you are not sure about the progress of any previous migration to encrypt data at rest,
+you must overwrite each resource with a new updated copy, in order to trigger encryption.
+See [Ensure all relevant data are encrypted](#ensure-all-secrets-are-encrypted) for details.
+{{< /note >}}
+
+## Encryption-at-rest configuration {#understanding-the-encryption-at-rest-configuration}
 
 {{< caution >}}
-**IMPORTANT:** For high-availability configurations (with two or more control plane nodes), the
-encryption configuration file must be the same! Otherwise, the `kube-apiserver` component cannot
-decrypt data stored in the etcd.
+For cluster configurations with two or more control plane nodes, the encryption configuration
+**must** be identical across each control plane node.
+
+If there is a difference in the encryption provider configuration, this may well mean
+that the kube-apiserver can't decrypt data stored inside the key-value store (potentially
+leading to further problems).
```

Comment:
Line 90-92 further elaborates the "must", but I failed to see a need for this paragraph.
This can be easily deduced.
We can elaborate the negative effects of such inconsistent settings, but obviously we cannot enumerate all cases.

Reply:
When we tell people that something **must** happen, it's good to provide context. This lets the reader evaluate whether to heed the advice.

For example, sometimes we write a **must** but all that happens if you break it is that some invalid object gets autoreplaced, and your cluster carries on. In another scenario, none of your ServiceAccounts work any more, the control plane degrades, and this is overall a very bad outcome. Context does help.